packages/data-stores/src/starter-designs-queries/use-starter-designs-query.ts

```diff
@@ -26,12 +31,14 @@ interface StaticDesign {
 	title: string;
 	categories: Category[];
 	price?: string;
+	style_variations?: ThemeStyleVariation[];
 }
 
 interface GeneratedDesign {
 	slug: string;
 	title: string;
 	recipe: DesignRecipe;
+	style_variations?: ThemeStyleVariation[];
```

Comment:
We're not returning `style_variations` for generated designs (at least for now) 😄

Reply:
Good catch, removed!